ElrondNetwork/elrond-go
# FIX: seed node could not connect to peers

## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
- When running a local testnet, seednode could not connect to any peer
  
## Proposed Changes
- Generated new seednode p2pkey in `seednode/config` and replaced it in other config files.
- When calling `copySeednodeConfig`, we also copy seednode key

## Testing procedure
- Run a local testnet with standard procedure:
1. Go to `scripts/testnet`
2. `./clean.sh`
3. `./config.sh`
4. `./start.sh`. Here, seednode should connect to all peers
5. `./stop.sh` when finished
